docs/LOCALHOST.md

```diff
@@ -71,3 +71,36 @@ delete from meadowlark.references;
 delete from meadowlark.aliases;
 delete from meadowlark.authorization;
 ```
+
+## Running E2E Tests
+
+### Setup
+
+1. Create a .env file based on .env.example in this folder.
+2. Verify that Docker is running
+
+There are multiple ways of running e2e tests, depending on your scenario:
+
+### Building local image
+
+If you're adding new features to Meadowlark, and want to test the changes in an isolated environment, run `npm run test:e2e:build`.
+This will build the Docker image and run the tests against that image, cleaning the environment afterwards.
+
+### Using published image
+
+If you want to run the tests against and existing Docker image, set the variable `API_IMAGE_NAME` in the .env file.
+Run the tests with `npm run test:e2e:jest`.
+This will pull the image and run the tests.
+
+### Developer Mode
```

Comment:
@stephenfuqua let me know if the details are clear.

I'm thinking if we should add an option to build just the backend in "dev-mode" that way we could then run the tests against an image that is being built constantly without clearing the backend. What are your thoughts?

Reply:
I don't understand.